app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -218,9 +217,37 @@ class Settings private constructor(
 
     var shouldUseBottomToolbar by booleanPreference(
         appContext.getPreferenceKey(R.string.pref_key_toolbar_bottom),
-        default = true
+        // Default accessibility users to top toolbar
+        default = !touchExplorationIsEnabled || !switchServiceIsEnabled
```

Comment:
I think you want `&&` here, no? Default to bottom when touch exploration is disabled **and** switch service is disabled.

Reply:
I literally spent like 5 minutes making sure this was correct and then still flipped the logic. Good catch 😄 thanks!